client/state/test/index.js

```diff
@@ -0,0 +1,46 @@
+require( 'lib/react-test-env-setup' )();
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+import mockery from 'mockery';
+
+describe( 'state', () => {
+	var createReduxStore;
+	before( () => {
+		mockery.registerMock( 'lib/themes/middlewares.js', {
+			// stub redux middleware
+			// see: http://rackt.org/redux/docs/advanced/Middleware.html
+			analyticsMiddleware: store => next => action => { return next( action ); }
+		} );
+		mockery.enable( {
+			warnOnReplace: false,
+			warnOnUnregistered: false
+		} );
+		createReduxStore = require( 'state' ).createReduxStore;
+	} );
+	after( () => {
+		mockery.deregisterAll();
+		mockery.disable();
+	} );
+	describe( 'createReduxStore', () => {
+		it( 'can be called without specifying initialState', () => {
+			const reduxStoreNoArgs = createReduxStore().getState();
+			const reduxStoreWithEmptyState = createReduxStore( {} ).getState();
+			const reduxStoreWithNullState = createReduxStore( null ).getState();
+			expect( reduxStoreNoArgs ).to.eql( reduxStoreWithEmptyState );
+			expect( reduxStoreWithNullState ).to.eql( reduxStoreWithEmptyState );
+		} );
```

Comment:
I like this test, though I'm left wondering just a bit what it actually confirms. It seems to confirm that in JavaScript the `undefined` and `null` are both _falsey_, since we have that default variable _and_ we aren't checking the value of the returned `getState()` here.

Is it possible to define an expected initial state here? If so, that would make this test more solid because right now we could end up returning `undefined` from these calls to `getState()` and this test should pass.

```js
const initialState = { none: { of: { your: 'business' } } };

[ undefined, {}, null ].forEach( state => expect( createReduxStore( state ).getState() ).to.eql( initialState ) );
```